flight/core/Dispatcher.php

```diff
@@ -23,41 +24,54 @@ class Dispatcher
 {
     public const FILTER_BEFORE = 'before';
     public const FILTER_AFTER = 'after';
-    private const FILTER_TYPES = [self::FILTER_BEFORE, self::FILTER_AFTER];
 
-    /** @var mixed $containerException Exception message if thrown by setting the container as a callable method */
-    protected $containerException = null;
+    /** Exception message if thrown by setting the container as a callable method. */
+    protected ?Exception $containerException = null;
```

Comment:
So I.....think this will work? This is an exception handled by the container exception. If they don't extend Exception for some reason, then this will fail. I just don't know what I don't know about how PHP-DI, Dice, Aura, et'c handle their exceptions.

Reply:
What is above exception is the throwable interface, unless the DI containers return Error instead of Exception, the property will accept the reassignment correctly

Reply:
All PSR/11 must either throw the exception required by the interface, the only thing that can fail is that the callable provided as the handler throws an Error and not an Exception

Reply:
I already changed it and uploaded it